src/application-insights/azext_applicationinsights/_params.py

```diff
@@ -27,6 +27,11 @@ def load_arguments(self, _):
     with self.argument_context('monitor app-insights component update-tags') as c:
         c.argument('tags', tags_type)
 
+    with self.argument_context('monitor app-insights component billing') as c:
+        c.argument('stop_send_notification_when_hit_cap', options_list=['-s', '--stop'], arg_type=get_three_state_flag(),
```

Comment:
stop_sending_notification_when_hitting_cap

Reply:
will change it.